.buildkite/metricbeat/pipeline.yml

```diff
@@ -30,6 +30,32 @@ env:
   TEST_COVERAGE: "true"
 
 steps:
+  - group: "Metricbeat Check/Update"
+    key: "metricbeat-check-update"
+    steps:
+      - label: "Run check/update"
+        command: |
+          make -C metricbeat check update
```

Comment:
I wonder, if this fails and the next command succeeds (assuming it's possible) the job would be successful right?
So, I think we want to fail fast, i.e. either add `set -eo pipefail` before the two `make` commands, or, instead:

`make -C metricbeat check update && make check-no-changes`

what do you think?

Reply:
Good point! How do you think, will be better to do it in separate PR for all beats?

Reply:
Change will be done for all beats in separate PR.